avaframe/ana3AIMEC/dfa2Aimec.py

```diff
@@ -209,11 +218,19 @@ def getPathsFromSimName(pathDict, avaDir, cfg, inputDirRef, simNameRef, inputDir
     suffix = ['pfd', 'ppr', 'pfv']
     for suf in suffix:
         refFile = inputDirRef / (simNameRef + '_' + suf + '.asc')
+        if not refFile.is_file():
+            message = 'No file found called: %s' % (str(refFile))
+            log.error(message)
+            raise FileNotFoundError(message)
         pathDict[suf].append(refFile)
-        log.info('Added to pathDict[%s] %s ' % (suf,refFile))
-        compFile =  inputDirComp / (simNameComp + '_' + suf + '.asc')
+        log.info('Added to pathDict[%s] %s ' % (suf, refFile))
+        compFile = inputDirComp / (simNameComp + '_' + suf + '.asc')
+        if not refFile.is_file():
```

Comment:
should this be `compFile`? Otherwise refFile is being checked twice

Reply:
right, compFile